pkg/system/xattrs_linux.go

```diff
@@ -1,35 +1,21 @@
 package system
 
-import (
-	"syscall"
-	"unsafe"
-)
+import "golang.org/x/sys/unix"
 
 // Lgetxattr retrieves the value of the extended attribute identified by attr
 // and associated with the given path in the file system.
 // It will returns a nil slice and nil error if the xattr is not set.
 func Lgetxattr(path string, attr string) ([]byte, error) {
```

Comment:
Just checked. The wrapper is still necessary, as there is also a version in pkg/system/xattrs_unsupported.go for !linux.